python/xnd/util.h

```diff
@@ -36,7 +36,12 @@
 
 
 #include <Python.h>
+#if PY_VERSION_HEX < 0x030B0000
+/*
+ * Python 3.11 moved longintrepr.h and it is already included in Python.h
```

Comment:
Could you also link a relevant PR/issue here?

Reply:
looks like this one: https://github.com/python/cpython/pull/28968 from browsing the repo.

Reply:
I'm no expert on CPython internals. If you're satisfied, please put it into the code comment and I'll merge the PR.

Reply:
done!